package.json

```diff
@@ -17,8 +17,11 @@
   "module": "./dist/bundle/module.js",
   "types": "./dist/declarations/index.d.ts",
   "exports": {
-    "import": "./dist/bundle/index.mjs",
-    "require": "./dist/bundle/index.js"
+    "./package.json": "./package.json",
+    ".": {
```

Comment:
are you sure on that one ? it seems awkward. Anyway I believe you should rather open an issue on Metro repo. This setup is pretty common, is not it?

Reply:
Pretty sure, as it fixes the issue. I believe Metro processes the `package.json` of dependent modules by importing them instead of reading the file directly. In such case, the `package.json`'s path must be specified in the `exports` field. It seems that there has been a lot of discussion around this issue, but not much of a conclusion I think.

https://github.com/react-native-community/cli/issues/1168
https://github.com/nodejs/node/issues/33460

Reply:
Ok 👍 . I was referring to the "."
Have you checked this path does not break nodejs cjs and es import and tools like rollup ?

Reply:
No, I haven't. I've only checked in React Native.

Reply:
yeah it should all work correctly, this is just the unsugared version of the previous and the fallbacks for old stuff (webpack 4, old nodes) is also correct.

refs
https://nodejs.org/dist/latest-v14.x/docs/api/packages.html#packages_determining_module_system
https://nodejs.org/dist/latest-v14.x/docs/api/packages.html#packages_dual_commonjs_es_module_packages
https://webpack.js.org/guides/package-exports/#reference-syntax

Reply:
With the new line for package.json we can't use the sugar syntax anymore and so needs to be unpacked to the verbose version.